lib/sync/installation.js

```diff
@@ -3,6 +3,10 @@ const getJiraClient = require('../jira/client');
 const { getRepositorySummary } = require('./jobs');
 const enhanceOctokit = require('../config/enhance-octokit');
 const statsd = require('../config/statsd');
+const { getLog } = require('../../lib/config/logger');
```

Comment:
Same as above. No need of getLog. instead pass `app` to `updateJobStatus` then app.log should work as it is